Sources/UnleashProxyClientSwift/UnleashProxyClientSwift.swift

```diff
@@ -33,13 +33,11 @@ public class UnleashClientBase {
     var poller: Poller
     var metrics: Metrics
 
-    public init(unleashUrl: String, clientKey: String, refreshInterval: Int = 15, metricsInterval: Int = 30, disableMetrics: Bool = false, appName: String = "unleash-swift-client", environment: String? = nil, poller: Poller? = nil, metrics: Metrics? = nil) {
+    public init(unleashUrl: String, clientKey: String, refreshInterval: Int = 15, metricsInterval: Int = 30, disableMetrics: Bool = false, appName: String = "unleash-swift-client", environment: String? = "default", context: [String: String]? = nil, poller: Poller? = nil, metrics: Metrics? = nil) {
```

Comment:
It's a bit confusing that fields like appName and environment can be provided as an argument and also as part of the initial context. Also, I noticed they're filtered out later. 

Given that anyone who wants to use this method will have to make code changes, why not add another initializer where all context properties are provided in this [String: String] map? So if you want to use it, you just use a different initializer

Reply:
In JS SDK we have those separated: https://github.com/Unleash/unleash-proxy-client-js/blob/main/src/index.ts#L25
Historically the context in swift was modelled as [String: String] and I want to change it in a major/breaking version in future. For now I want parity between init and updateContext and updateContext already works this way. appName is required anyway. I also want to keep the API surface area minimal and not add anything.

Reply:
Also added a README entry about it

Reply:
Pair programmed this part :+1: